odl/operator/default_ops.py

```diff
@@ -79,36 +79,19 @@ def _apply(self, x, out):
         >>> vec = r3.element([1, 2, 3])
         >>> out = r3.element()
         >>> op = ScalingOperator(r3, 2.0)
-        >>> op(vec, out)  # Returns out
+        >>> op(vec, out)  # In place, Returns out
         Rn(3).element([2.0, 4.0, 6.0])
         >>> out
         Rn(3).element([2.0, 4.0, 6.0])
-        """
-        out.lincomb(self._scal, x)
-
-    def _call(self, x):
-        """Return the scaled element.
-
-        Parameters
-        ----------
-        x : domain element
-            input vector to be scaled
-
-        Returns
-        -------
-        scaled : ``range`` element
-            The scaled vector
-
-        Examples
-        --------
-        >>> from odl import Rn
-        >>> r3 = Rn(3)
-        >>> vec = r3.element([1, 2, 3])
-        >>> op = ScalingOperator(r3, 2.0)
-        >>> op(vec)
+        >>> op(vec)  # Out of place
         Rn(3).element([2.0, 4.0, 6.0])
         """
-        return self._scal * x
+        if out is None:
+            out = x.copy()
```

Comment:
Simply `out = self._scal * x` does the same thing but is more brief and could allow some duck-typing.